tareas/clase-5/tarea3mio.js

```diff
@@ -0,0 +1,53 @@
+//TAREA: En otro archivo distinto,
+// Crear una lista de <ol> y <li> que contengan sólo números.
+// Convertir esos números a un array y:
+// 1. calcular el promedio y mostrarlo en un <em> pre-creado con el texto "El promedio es..."
+// 2. obtener el número más pequeño y mostrarlo en un <em> pre-creado con el texto "El número más pequeño es..."
+// 3. obtener el número más grande y mostrarlo en un <em> pre-creado con el texto "El número más grande es..."
+// 4. obtener el número que más se repite y mostrarlo en un <em> pre-creado con el texto "El número más frecuente es..."
+
+let nodoLi = document.querySelectorAll(".lista");
+let textoEm1 = document.querySelector(".em1");
+let textoEm2 = document.querySelector(".em2");
+let textoEm3 = document.querySelector(".em3");
+let textoEm4 = document.querySelector(".em4");
+let listaArray = [];
+
+for (let i = 0; nodoLi.length > i; i++) {
+	listaArray = listaArray + nodoLi[i].innerText;
+}
+
+function calcularPromedio() {
+	let acumulador = 0;
+	for (let i = 0; listaArray.length > i; i++) {
+		acumulador = acumulador + Number(listaArray[i]);
+	}
+	acumulador = acumulador / listaArray.length;
+	let $acumulador = document.createTextNode(acumulador);
+	textoEm1.appendChild($acumulador);
+}
+calcularPromedio();
+
+function calcularMayor() {
+	let acumulador = 0;
+	for (let i = 0; listaArray.length > i; i++) {
+		if (listaArray[i] > acumulador) {
+			acumulador = listaArray[i];
+		}
+	}
+	let $acumulador = document.createTextNode(acumulador);
+	textoEm2.appendChild($acumulador);
+}
+calcularMayor();
+
```

Comment:
una vez tengas `listaArray` aqui lo que quieres es que la función haga lo que dice que hace solamente, (en este caso calcular el numero mayor)entonces una forma de organizar esta y las funciones `calcularPromedio` y `calcularMenor` es usar `return` con el valor que quieres que la función te devuelva, asi: 

`function calcularMayor() {
  let acumulador = 0;
  for (let i = 0; i < listaArray.length; i++) {
    if (listaArray[i] > acumulador) {
      acumulador = listaArray[i];
    }
  }
  return acumulador;
}

let $acumulador = document.createTextNode(calcularMayor(listaArray));
textoEm2.appendChild($acumulador);`